emscripten/1_download_library.sh

```diff
@@ -16,22 +16,26 @@ msg " [1] Preparing Emscripten SDK"
 nproc=$(nproc)
 export MAKEFLAGS="-j${nproc:-2}"
 
-download_and_extract https://s3.amazonaws.com/mozilla-games/emscripten/releases/emsdk-portable.tar.gz
+rm -rf emsdk-portable
+git_clone https://github.com/emscripten-core/emsdk.git emsdk-portable
 cd emsdk-portable
 
 # Fetch the latest registry of available tools.
-./emsdk update
+./emsdk update-tags
 
 # Download and install the latest SDK tools and set up the compiler configuration to point to it.
-./emsdk install sdk-incoming-64bit
-./emsdk activate sdk-incoming-64bit
+./emsdk install latest-upstream
+./emsdk activate latest-upstream
```

Comment:
I wanted to use 1.38.46 (latest release) here but when I do this libpng can't find "pow"